src/test/java/kg/jarkyn/cobspecserver/responders/PublicResourceResponderTest.java

```diff
@@ -44,6 +44,15 @@ public void respondsWithDirectoryListing() {
         assertEquals(directoryListingResponse(), response.getContent());
     }
 
+    @Test
+    public void respondsWithNotFound() {
```

Comment:
I find these tests too low-level. Overall the intention is clear: when a url that does not map to a resource is hit, 404 is returned. That makes sense, however this test is matching the _exact_ http response that is returned. Seeing the HTTP version and a bunch of carriage returns in the assertions is distracting and lower-level than the rest of the test.

The assertion of this test could read like this:
```java
 assertEquals("404", response.getStatus());
```

Or, with an enum:
```java
 assertEquals(Status.NOT_FOUND, response.getStatus());
```